main.js

```diff
@@ -1284,23 +1284,22 @@ function main() {
     });
 
     zwave.on('value removed', function (nodeID, comClass, instance, index) {
-        adapter.log.info('value removed: ' + nodeID + ' comClass: ' + JSON.stringify(comClass) + ' instance ' + instance + ' value: '  + JSON.stringify(index));
+        adapter.log.debug('value removed: ' + nodeID + ' comClass: ' + JSON.stringify(comClass) + ' instance ' + instance + ' value: '  + JSON.stringify(index));
 
-        deleteDevice(nodeID, function() { //why here and not on "node removed" ?
-            //disable exclusion automatically after 5 seconds
-            setTimeout(function(){disableExclusion();}, 5000);
-        });
+        var id = calcName(nodeID, comClass, index, instance);
```

Comment:
We should use `let` and `const` moving forward. So this should be `const id = ...`.